--- lib/gitlab_config.rb.old	2014-08-21 05:44:03.144297001 +0200
+++ lib/gitlab_config.rb	2014-08-21 05:49:49.992296992 +0200
@@ -53,7 +53,7 @@
       if redis.has_key?("socket")
         %W(#{redis['bin']} -s #{redis['socket']})
       else
-        %W(#{redis['bin']} -h #{redis['host']} -p #{redis['port']})
+        %W(#{redis['bin']} -h #{redis['host']} -p #{redis['port']} -n #{redis['db']})
       end
     end
   end
--- config.yml.example.old	2014-08-21 05:49:59.672297001 +0200
+++ config.yml.example	2014-08-21 05:50:15.848678932 +0200
@@ -29,6 +29,7 @@
   bin: /usr/bin/redis-cli
   host: 127.0.0.1
   port: 6379
+  db: 0
   # socket: /tmp/redis.socket # Only define this if you want to use sockets
   namespace: resque:gitlab
 
