--- hatari-1.9.0.orig/python-ui/uihelpers.py	2015-09-19 19:55:21.000000000 +0200
+++ hatari-1.9.0/python-ui/uihelpers.py	2015-09-19 20:01:01.000000000 +0200
@@ -98,9 +98,7 @@
         # first try whether there are local Hatari docs in standard place
         # for this Hatari/UI version
         sep = os.sep
-        path = self.get_binary_path("hatari")
-        path = sep.join(path.split(sep)[:-2]) # remove "bin/hatari"
-        path = path + sep + "share" + sep + "doc" + sep + "hatari" + sep
+        path = "@DOCDIR@"
         if os.path.exists(path + "manual.html"):
             return path
         # if not, point to latest Hatari HG version docs
