--- src/gpodder/youtube.py.orig	2014-05-11 16:20:58.000000000 +0200
+++ src/gpodder/youtube.py	2014-05-11 16:21:10.000000000 +0200
@@ -116,7 +116,7 @@
                 fmt_url_map = urllib.unquote(r4.group(1))
                 for fmt_url_encoded in fmt_url_map.split(','):
                     video_info = parse_qs(fmt_url_encoded)
-                    yield int(video_info['itag'][0]), video_info['url'][0] + "&signature=" + video_info['sig'][0]
+                    yield int(video_info['itag'][0]), video_info['url'][0]
             else:
                 error_info = parse_qs(page)
                 error_message = util.remove_html_tags(error_info['reason'][0])
