--- config/environments/production.rb.old	2014-08-21 04:25:35.769297038 +0200
+++ config/environments/production.rb	2014-08-21 04:25:45.933297037 +0200
@@ -75,11 +75,10 @@
   config.active_support.deprecation = :notify
 
   config.action_mailer.delivery_method = :sendmail
-  # Defaults to:
-  # # config.action_mailer.sendmail_settings = {
-  # #   location: '/usr/sbin/sendmail',
-  # #   arguments: '-i -t'
-  # # }
+  config.action_mailer.sendmail_settings = {
+    location: '/usr/sbin/sendmail',
+    arguments: '-i'
+  }
   config.action_mailer.perform_deliveries = true
   config.action_mailer.raise_delivery_errors = true
 
